pipeline/api/cloud.py

```diff
@@ -308,64 +308,34 @@ def upload_pipeline_file(
         )
 
     def _get(self, endpoint: str, params: Dict[str, Any] = None):
-        headers = {
-            "Authorization": "Bearer %s" % self.token,
-        }
-
-        url = urllib.parse.urljoin(self.url, endpoint)
-        response = httpx.get(url, headers=headers, params=params, timeout=self.timeout)
+        self.raise_for_invalid_token()
```

Comment:
@paulcjh I noticied that `_get` was unique in not having `raise_for_invalid_token`. I added it in case it was an oversight, but perhaps it's missing intentionally?

Reply:
No, nice spot!